packages/tools/pie-icons-react/icons/IconEggsLarge.tsx

```diff
@@ -0,0 +1,20 @@
+import * as React from "react";
+import { LargeIconProps } from "../types";
+// @ts-ignore
+import { getSvgProps } from './configs-react';
+const IconEggsLarge = (props: LargeIconProps) => {
+  const {
+    className,
+    iconSize,
+    width,
+    height,
+    ...remainingProps
+  } = props;
+  const moreProps = getSvgProps("pie-icon pie-icon--eggs-large", className, iconSize, "IconEggsLarge");
```

Comment:
try to name this after the method. `getSvgProps` gets `svgProps` (or `commonSvgProps` in which case you'd probably rename that method too).

`...remainingProps` is commonly referred to as `...rest`.

Reply:
In principle I agree, however these `tsx` and `js` files are generated code so not something I'd like to change as part of this PR.

Reply:
Understood. Perhaps something to track though?

Reply:
@webbertakken it's a fair point! I think as mentioned above changes to the auto-generated files are out of scope for this and I don't think the PR thread is the best place to track it. Would you be happy to raise a discussion in our team channel or a GH discussion? Would love to see how we might improve some of this stuff :)